proto-lens-protoc/src/Definitions.hs

```diff
@@ -164,6 +165,33 @@ fieldName = unpack . disambiguate . camelCase
                 [""] -> error "camelCase: name consists only of underscores"
                 s':ss -> T.concat $ underlines : toLower s' : map capitalize ss
 
+-- | A list of reserved keywords in Haskell2010.
+reservedIds :: Set.Set Text
```

Comment:
Maybe rename to reservedKeywords? Does GHC expose this list by chance? Or would that bring in a large dependency?